sqlmesh/core/model/definition.py

```diff
@@ -1918,10 +1925,11 @@ def _create_model(
     physical_schema_mapping: t.Optional[t.Dict[re.Pattern, str]] = None,
     python_env: t.Optional[t.Dict[str, Executable]] = None,
     audit_definitions: t.Optional[t.Dict[str, ModelAudit]] = None,
-    default_audits: t.Optional[t.List[AuditReference]] = None,
+    default_audits: t.Optional[t.List[FunctionCall]] = None,
     inline_audits: t.Optional[t.Dict[str, ModelAudit]] = None,
     module_path: Path = Path(),
     macros: t.Optional[MacroRegistry] = None,
+    signal_definitions: t.Optional[SignalRegistry] = None,
```

Comment:
Shouldn't this be a part of the metadata hash just like audits?

Reply:
it should be already

Reply:
modified it and will add a test, signals were already a part of metadata, and python_env is as well